site/en/docs/privacy-sandbox/shared-storage/index.md

```diff
@@ -19,11 +19,9 @@ Storage API.
 
 *  The [Shared Storage proposal](https://github.com/WICG/shared-storage)
    has entered [public discussion](https://github.com/WICG/shared-storage/issues).
-*  We're [implementing this API in Chrome version 104](#try-the-shared-storage-api).
-   *  The initial origin trial will only include the
-      `runURLSelectionOperation`. Support for the Private Aggregation API will
-      come shortly after.
-   *  The `selectURL` API is available for testing in Chrome Canary as of M104.
+*  Only the URL Selection (`selectURL`) output gate is available for testing at the moment. Support for the Private Aggregation API will come soon.
+*  The API is available for [Privacy Sandbox unified origin trial](/blog/expanding-privacy-sandbox-testing/) on Chrome Canary/Dev/Beta M105 or later.
```

Comment:
```suggestion
*  The API is available in the [Privacy Sandbox unified origin trial](/blog/expanding-privacy-sandbox-testing/)
   on Chrome Canary/Dev/Beta M105 or later.
*  Only the URL Selection (`selectURL`) output gate is available for
   testing at the moment. Support for the Private Aggregation API will
   come soon.
```

Reply:
Reversed these two points, as the API availability for testing is more important than what can be tested.